src/components/ConfigureSupersetEmbeddedDashboardModal/CreateSupersetEmbeddedDashboard.js

```diff
@@ -46,7 +31,9 @@ export const CreateSupersetEmbeddedDashboard = ({
     const dispatch = useDispatch()
     const history = useHistory()
     const [loading, setLoading] = useState(false)
-    const [postDashboard, { error }] = useDataMutation(postDashboardQuery)
+    const [postDashboard, { error }] = useDataMutation(postDashboardQuery, {
+        onError: () => setLoading(false),
+    })
```

Comment:
This change is not directly related to the current work, but I realised that in case of an error we would end up both in the error and the loading state here.